utils/common/src/test/java/org/apache/brooklyn/util/osgi/OsgiTestResources.java

```diff
@@ -103,5 +103,14 @@
      * at runtime.
      */
     public static final String BROOKLYN_TEST_MORE_ENTITIES_V2_EVIL_TWIN_PATH = "/brooklyn/osgi/" + BROOKLYN_TEST_MORE_ENTITIES_SYMBOLIC_NAME_FINAL_PART + "_evil-twin_0.2.0.jar";
+
+
+    public static final String BROOKLYN_TEST_OSGI_ENTITIES_URL = "classpath:"+OsgiTestResources.BROOKLYN_TEST_OSGI_ENTITIES_PATH;
```

Comment:
personal preference to keep it in case you don't notice it's an interface; don't feel strongly but not going to rewrite this class in this PR